site/e2e/server/server.mjs

```diff
@@ -0,0 +1,4 @@
+import { chromium } from "@playwright/test";
+
+const server = await chromium.launchServer({ headless: false });
```

Comment:
I suspect that .mjs extension is on purpose?

Reply:
When running in Node, this was the only way I could make `import`-statements work (vs `require`), I also tried `tsnode` with `.ts` but that was a bust. Open for alt. suggestions.